plugins/UFPWriter/UFPWriter.py

```diff
@@ -74,6 +77,19 @@ def write(self, stream, nodes, mode = MeshWriter.OutputMode.BinaryMode):
             Logger.error(error_msg)
             return False
 
+        #Write settings
+        try:
+            archive.addContentType(extension="json", mime_type="application/json")
+            setting_textio = StringIO()
+            json.dump(self._getSettings(), setting_textio, separators=(", ", ": "), indent=4)
```

Comment:
Do we need `indent=4`. might increase file size, and this is only going to be read by programs.

Reply:
It would make it easier to read. ZIP will probably compress that away very effectively.